### ISSUE TITLE
Constructor comments are not handled correctly

### ISSUE THREAD

**Issue by barakman** [soldocjs/soldoc#17]
* **Version**:v8.2.1
* **Platform**:64-bit (Windows)
* **Subsystem**:

Documentation generation fails when constructor comments are missing.

In file `pdf.js` line 55, I had to extend this:
```
if (contract.contractData.constructor !== null)
```
To this:
```
if (contract.contractData.constructor !== null && contract.contractData.constructor.comments !== undefined)
```

Thanks

**Comment by obernardovieira** [comment]
Hello,
Thank you so much, I'll have a look.
BTW. I will release a new version with a new PDF template within 2~3 days.

**Comment by barakman** [comment]
Thanks @obernardovieira.

A couple of note:
1. The generated output includes the tag names (preceded by `@`). I believe that this is pretty undesirable.
2. I was actually asked to find a tool which generates an output in MD format, in order to allow us to publish it on GitBook. Your tool seems to be very easy to set up and use, so it would be very nice if you could add support for MD format.

**Comment by barakman** [comment]
Thanks @obernardovieira.

A couple of note:
1. The generated output includes the tag names (preceded by `@`). I believe that this is pretty undesirable.
2. I was actually asked to find a tool which generates an output in MD format, in order to allow us to publish it on GitBook. Your tool seems to be very easy to set up and use, so it would be very nice if you could add support for MD format.

**Comment by obernardovieira** [comment]
Great.
1. I agree it was just a first draft (a little better next release).
2. Actually, I was initially generating the PDF by generating an MD first and then HTML and then PDF. But I decided to remove the MD step and generate the HTML using the same template engine as the webpage. And there was a moment when I was considering between a PDF and a gitbook. So yes, your idea is very doable. Please, open a new issue and describe that.

**Comment by obernardovieira** [comment]
Fixed. Will be released with 0.1.0
